spezi-invitation-code/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "spezi-invitation-code",
+  "version": "1.0.0",
+  "description": "Invitation code verifier for Spezi Firebase",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
We should add at least some tests + linting in this repo. We can use the same firebase setup (that seeds the DB with content) as we use in the other repos to test the invitation code verification and we could use a simple JS script that makes the function calls / creates a user using the firebase SDK (only imported for the tests).

Regarding linting and some surrounding setups, I would suggest to look at the NextJS template setup that we have over here: https://github.com/StanfordBDHG/NextJSTemplate/blob/main/package.json. We don't have to include all the NextJS setups but all the GitHub Actions, linting, and testing setup might be something that we can transfer over here ...